.jenkins/pod/build_container.yml

```diff
@@ -0,0 +1,10 @@
+metadata:
+  annotations:
+    iam.amazonaws.com/role: ecr-push-pull
```

Comment:
Why do we not use the service account here?

I understand that we cannot migrate everything, but I think we should not add new stuff using kube2iam when we know we want to deprecated it.

Reply:
Because I need to check first the best way to use the service account in the current Kubernetes plugin. For the time being, I think makes sense to keep everything as is.

Reply:
Not sure I get it but let's move forward ;)